tracks/cpp/exercises/grains/mentoring.md

```diff
@@ -0,0 +1,45 @@
+# Mentoring
+
+## First function
+
+The first way to implement this function is to use a for loop :
+```cpp
+unsigned long long square(int indice) {
+    unsigned long long result = 1;
+    for (int i = 0; i < indice; i++) result *= 2;
+    return result;
+}
+```
+A cleaner way to do this is to use a recursive function :
```

Comment:
I'd argue that it is subjective whether a recursive implementation is "cleaner" than one using a loop.  
*Personally* I prefer the function with the loop, with my imperative mindset I find it a tiny bit easier to understand, and it avoids the overhead of the recursive function call.  
If you really want to call it "cleaner" I'd prefer if you make it clear that this is an opinion (like later with the "in my mind".)